### PR TITLE
Don't apply impl block collapse rules to trait impls

### LINKED ISSUE #58147: [rustdoc] Collapsing everything doesn't collapse trait impls content

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #58147.

r? @QuietMisdreavus

### CODE REVIEW

File: src/librustdoc/html/static/main.js [hunk 1]
```diff
@@ -1974,7 +1974,7 @@ if (!DOMTokenList.prototype.remove) {
                 }
             }
         } else {
-            // we are collapsing the impl block
+            // we are collapsing the impl block(s?).
```
Comment (Manishearth) [inline src/librustdoc/html/static/main.js]: remove the question mark

**Review by Manishearth — APPROVED** [review]
r=me

File: src/librustdoc/html/static/main.js (hunk 1 above)
Comment (GuillaumeGomez) [reply src/librustdoc/html/static/main.js]: Done.

**Pull request merged**